src/main/g8/app/pages/QuestionPage.scala

```diff
@@ -0,0 +1,29 @@
+package pages
+
+import controllers.routes
+import models.{CheckMode, Mode, NormalMode}
+import play.api.mvc.Call
+import utils.UserAnswers
+
+trait QuestionPage[A] extends Page {
+
+  def cleanup(value: Option[A], userAnswers: UserAnswers): UserAnswers = userAnswers
```

Comment:
Method cleanup take `value` and `userAnswers`. Is it correct to return `userAnswers` without any change?

Reply:
This is the default implementation (because on most pages, when we save the answer we don't want to change any others).  Where you _do_ want to do some cleanup, you'd override this method on that specific page.

https://github.com/TheNickWilson/examples-pages-frontend/blob/master/app/pages/LocationPage.scala#L29-L32 is an example of this